### DISCUSSION TITLE
翻译 PR 定期出现这种情况

### DISCUSSION THREAD

**Discussion by Usualminds** [xitu/gold-miner#8997] [Q&A]
@lsvih @itcodes @PassionPenguin 翻译 PR 列表定期会出现情况，是在测试什么吗？感觉对 PR 认领干扰很大，https://github.com/xitu/gold-miner/pulls

![image](https://user-images.githubusercontent.com/19425902/129197862-80fe0ce8-f78c-453d-8611-8b5177a757ce.png)

**Comment by lsvih** [comment]
收到，今天及时处理

**Comment by PassionPenguin** [comment]
唔优弧让我自己翻然后推上来，管理员之间扯皮 - -
